apitools/base/py/credentials_lib.py

```diff
@@ -384,6 +387,21 @@ def _refresh(self, _):
     self.access_token = token
 
 
+def _GetRunFlowFlags():
+  if FLAGS is None:
```

Comment:
given that i voted for allowing a mix of `flags` and `argparse` below, probably better safe than sorry.

feel free to leave a `TODO` for me to clean this up when i drop `flags` completely.